PyTorch/vLLM_Tutorials/Deploying_vLLM/README.md

```diff
@@ -279,6 +263,40 @@ docker run -it --rm \
 docker logs -f vllm-server
 ```
 
+# Running vLLM server with **DTYPE=fp8**.
+
+1) Example for bringing up a vLLM server with a custom max model length, tensor parallel (TP) size and **DTYPE=fp8**.
+
+1.1) Run the command below to create measurement_version.txt which captures the current version of the repository. 
+
+```bash
+bash get_version.sh
+```
+
+1.2) Run below command to bring up the server
+
+```bash
+docker run -it --rm \
+    -e http_proxy=$http_proxy -e https_proxy=$https_proxy -e no_proxy=$no_proxy \
+    -e HF_HOME=/mnt/hf_cache \
+    -v /mnt/hf_cache:/mnt/hf_cache \
+    --cap-add=sys_nice \
+    --ipc=host \
+    --runtime=habana \
+    -e HF_TOKEN=YOUR_TOKEN_HERE \
+    -e HABANA_VISIBLE_DEVICES=all \
+    -p 8000:8000 \
+    -e MODEL=meta-llama/Llama-3.1-70B-Instruct \
+    -e TENSOR_PARALLEL_SIZE=4 \
+    -e DTYPE=fp8 \
+    -e MAX_MODEL_LEN=8192 \
+    -v ./measurement:/root/scripts/measurement \
+    --name vllm-server \
+    vllm-v0.7.2-gaudi-ub24:1.21.1-16
+```
+- FP8 inference requires model statistics measurements see [docs.habana.ai - Run Inference using FP8](https://docs.habana.ai/en/latest/PyTorch/Inference_on_PyTorch/Quantization/Inference_Using_FP8.html). In the above example model measurement files are automatically generated and stored inside the container at /root/scripts/measurement.
+- Persistent measurement folder can also be provided, for example by adding docker volume using command line option **-v ./measurement:/root/scripts/measurement** as given in the above example, then statistics measurement will be skipped if measurement file already exists.
+
```

Comment:
Below in RAMDISK example change MODEL_CACHE_DIR=Llama-3.1-8B-Instruct_TP1_G2_$DTYPE change G2 to G3

Reply:
Since we have said **User can provide DTYPE in server command for good practice. It is left as user discretion** 
and now we that support fp8,. lets add
DTYPE=bfloat16
and 
-e DTYPE=$DTYPE \
in all the recipe cache cmdlines